<launch>
  <node pkg="ondo" name="tr7nw" type="tr7nw_pub.py" required="true">
    <remap from="tr7nw" to="tr7nw_1" />
    <param name="~host" value="192.168.100.200" />
    <param name="~serial_number" value="52181d25" />
    <param name="~rate" value="1" />
  </node>
</launch>